src/igvfd/schemas/modification.json

```diff
@@ -92,6 +95,18 @@
             "description": "The product or catalog identifier provided following deposition to addgene.org.",
             "type": "string",
             "pattern": "^addgene:\\d{5,6}$"
+        },
+        "cas_species": {
+            "title": "Cas species",
+            "description": "The originating species of the Cas nuclease.",
+            "type": "string",
+            "default": "Streptococcus pyogenes (Sp)",
```

Comment:
if it is required, not sure we should default to anything

Reply:
makes sense, i can remove it